belt-dwp/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "belt-dwp"
+version = "0.1.0"
+description = "Pure Rust implementation of the Belt-DWP authenticated encryption algorithm (STB 34.101.31-2020)"
+edition = "2024"
+license = "Apache-2.0 OR MIT"
+readme = "README.md"
+documentation = "https://docs.rs/belt-dwp"
+repository = "https://github.com/RustCrypto/AEADs/tree/master/belt-dwp"
+keywords = ["aead", "belt-dwp"]
+categories = ["cryptography", "no-std"]
+rust-version = "1.85"
+
+[dependencies]
+aead = { version = "0.6.0-rc.0", default-features = false }
+zeroize = { version = "1.7", default-features = false, optional = true }
+universal-hash = { version = "0.6.0-rc.0" }
+opaque-debug = { version = "0.3" }
+
+belt-block = { version = "0.2.0-pre.2" }
+belt-ctr = { version = "0.2.0-pre" }
+
+[dev-dependencies]
+hex-literal = "0.4"
+
+[features]
+default = ["alloc", "os_rng"]
+alloc = ["aead/alloc"]
+arrayvec = ["aead/arrayvec"]
+bytes = ["aead/bytes"]
+os_rng = ["aead/os_rng", "rand_core"]
+heapless = ["aead/heapless"]
+rand_core = ["aead/rand_core"]
+reduced-round = []
+zeroize = ["dep:zeroize", "belt-ctr/zeroize"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
We don't need to pass `--cfg docsrs` anymore, since docs.rs passes this flag itself by default (we should remove it from other crates as well). Also, keep an empty last line.

Reply:
You should keep `all-features = true`.

Reply:
Fixed